tools/docker/builder/openwrt/build.sh

```diff
@@ -42,11 +43,13 @@ build_prplmesh() {
            -v "$scriptdir/scripts:/home/openwrt/openwrt_sdk/build_scripts/:ro" \
            -v "${rootdir}:/home/openwrt/prplMesh_source:ro" \
            "$image_tag" \
-           ./build_scripts/build.sh
+           ./build_scripts/build.sh || status=$((status + 1))
 
-    docker cp "${container_name}:/home/openwrt/openwrt_sdk/prplmesh-${TARGET}-${OPENWRT_VERSION}-${PRPLMESH_VERSION}.ipk" .
+    docker cp "${container_name}:/home/openwrt/openwrt_sdk/prplmesh-${TARGET}-${OPENWRT_VERSION}-${PRPLMESH_VERSION}.ipk" . || status=$((status + 1))
 
-    docker rm "${container_name}"
+    docker rm -f "${container_name}"
```

Comment:
Isn't it easier and more appropriate to move this to a `trap EXIT` statement just before docker is started?

Reply:
Done, and force-pushed, I will rebase #628 as well.